src/api/v1/community_businesses/temporary/get.ts

```diff
@@ -0,0 +1,41 @@
+import * as Hapi from 'hapi';
+import { isChildOrganisation } from '../../prerequisites';
+import { response } from '../schema';
+
+export default [
+  {
+    method: 'GET',
+    path: '/community-businesses/temporary',
+    options: {
+      description: 'Retrieve list of all temporary community businesses',
+      auth: {
+        strategy: 'standard',
+        access: {
+          scope: ['organisations_details-child:read'],
+        },
+      },
+      pre: [
+        { method: isChildOrganisation, assign: 'isChild', failAction: 'error' },
+      ],
+      response: { schema: response },
+    },
+    handler: async (request: Hapi.Request, h: Hapi.ResponseToolkit) => {
+
+      const {
+        server: { app: { knex } },
+      } = request;
+
+      // TODO: stick in model method
```

Comment:
Let's actually do this?

Reply:
`CommunityBusinesses.getTemporary`?

Also do we want a separate type for temporary CB? Or happy for them to squeeze into a partial cb?

Reply:
`getTemporary` sounds fine. My initial reaction is that they don't need a separate type.

Reply:
So, technically, the return value does not match an array of our `CommunityBusiness` type. I thought this was okay because all the values I am not returning here are ones we are leaving empty eg address1, postcode ect

I feel like it's inaccurate in this instance to say that the model method will return `[] Partial<Cb>`, I could pick the values I know it'll return but it's very specific to this use case.

🤷🏽‍♀️ any thoughts? I was keen to leave this in the handler until this feature becomes a bit better understood (and whether we add anything similar to help with onboarding/analytics)

Reply:
I'm thinking that I'd like the models to be a place where we keep all the "questions" we are asking of the data. If they're all (roughly) in one place then it's easier to see patterns as and when they arise.

I'm currently not so bothered if the type signature of a few of the methods are super-specific.